module3/projects/take_home/tea_take_home_be.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Tea Subscription Service
+subheading: Take-Home Back End Prompt
+layout: page
+---
+
+## Things we'll be looking for:
+
+- A strong understanding of Rails
+- Ability to create restful routes
+- Demonstration of well-organized code, following OOP
+- Test Driven Development
+- Clear documentation
+
+Try to limit your total time to approximately 8 hours on this portion of the project. Prioritize what is important to completing MVP and demonstrating your capabilities as a developer.
+
+
+## Tea Subscription Service
+
+You will create a Rails API for a Tea Subscription Service.
+
+### Requirements:
+
+At a minimum, you must create:
+
+- An endpoint to subscribe a customer to a tea subscription
+- An endpoint to cancel a customer's tea subscription
```

Comment:
We'll probably want to change these to make them work for the FE, so maybe:
- get all teas
- get info about one tea
- subscribe a customer to a tea subscription
- get all customer's subscriptions